src/main/java/pl/touk/sputnik/connector/ConnectorFacadeFactory.java

```diff
@@ -15,6 +16,7 @@
     StashFacadeBuilder stashFacadeBuilder = new StashFacadeBuilder();
     GithubFacadeBuilder githubFacadeBuilder = new GithubFacadeBuilder();
     SaasFacadeBuilder saasFacadeBuilder = new SaasFacadeBuilder();
+    LocalFacadeBuilder localFacadeBuilder = new LocalFacadeBuilder();
```

Comment:
\[Checkstyle\] ERROR: Variable 'localFacadeBuilder' must be private and have accessor methods\.